tests/ha/clvmd_lvmlockd.pm

```diff
@@ -21,9 +21,10 @@ sub run {
     my $lvm_conf = '/etc/lvm/lvm.conf';
     my $lock_mgr = 'clvm';
 
-    # lvmlockd is only available in SLE15+
+    # Only SLE-15+ has support for lvmlockd
+    set_var('USE_LVMLOCKD', 0) if (get_var('USE_LVMLOCKD') and is_sle('<15'));
+
     if (get_var("USE_LVMLOCKD")) {
```

Comment:
Here the logic was:
"If a user externally (in the JobGroup) try to set `USE_LVMLOCKD` in a test not for 15 --> FAILS at it is not supported"

So configuration was outside the test code, validation only was within the test code

Reply:
Yes.